n-heptane data

bradcarr23 opened the pull request:
This is a new branch for the n heptane data, with the methyl decanoate data removed from the root directory. I believe that all previous errors should be corrected.

bryanwweber commented:
@bradcarr23 Sorry, looks like the MD data still made it in here: 2f9a23d Can you come see me in office hours to sort this out?

bradcarr23 commented:
@bryanwweber sure thing, or we can talk tomorrow before or after class. I believe that the latest commit removed the 1st commit which added the data in the first place. I thought that would have gotten rid of the MD data, sorry for the delay.

bryanwweber commented:
@bradcarr23 No problem, we'll just need to squash those commits together so its like the data was never added in the first place. I don't think we'll have time around class tomorrow, office hours would be better.

bradcarr23 commented:
@bryanwweber okay, I'll see you around 3:30 on Wednesday then!

bradcarr23 committed:
Addition of n-heptane data

Addition of n-heptane data, converted from xml format to yaml files

bryanwweber commented:
@bradcarr23 Can you add yourself to the `file-authors` as well?
```
file-authors:
  - name: Kyle Niemeyer
    ORCID: 0000-0003-4425-7097
  - name: Bradley Carrano
```
and edit the `chemked-version` to be 0.4.1?

bradcarr23 committed:
Updated files for N-Heptane

Ive added my name to the file-authors, as well as updated the chemked version to 0.4.1. I am going to begin converting the rest of the files to YAML by hand, as there have been errors using the convert function in ChemKED.

bryanwweber commented on n-heptane/Burcat 1981/st_burcat_1981-1.yaml:
```diff
@@ -1,7 +1,8 @@
-file-author:
+file-authors:
     name: Kyle Niemeyer
+    name: Bradley Carrano
```
I didn't notice this yesterday, but there should be a dash in front of each of these,
```
  - name: Kyle Niemeyer
  - name: Bradley Carrano
```

bradcarr23 committed:
Dashes added before authors

I also added my ORCID to these files. I'm not sure why there is a blank line before the file-version, as this is not in the files on my computer. Hopefully this isnt an issue!

bryanwweber commented:
@bradcarr23 Is this PR done now? There is a new one (#18) that works on these files, so we need to finish up ASAP.

bradcarr23 committed:
Fixed N-Heptane Data

Fixed formatting issues, added equivalence ratios and missing uncertainties, added missing ORCID
